### PR TITLE
SAK-32557    commons-lang3,  delete:commons-collections

### PR DESCRIPTION
**Pull request by axxter99**
import org.apache.commons.lang.StringEscapeUtils; => import org.apache.commons.lang3.StringEscapeUtils;
delete pom.xlms: <dependency>
<groupId>commons-collections</groupId>
<artifactId>commons-collections</artifactId>

### CODE REVIEW

File: polls/api/pom.xml [hunk 1]
```diff
@@ -30,9 +30,9 @@
             <groupId>org.sakaiproject.kernel</groupId>
             <artifactId>sakai-component-manager</artifactId>
         </dependency>
-        <dependency>
-            <groupId>commons-lang</groupId>
-            <artifactId>commons-lang</artifactId>
-        </dependency>
+		<dependency>
```
Comment (buckett) [inline polls/api/pom.xml]: Rest of the file looks to use spaces for indentation rather than tabs.

File: polls/tool/pom.xml [hunk 2]
```diff
@@ -42,10 +42,10 @@
             <groupId>org.sakaiproject.kernel</groupId>
             <artifactId>sakai-component-manager</artifactId>
         </dependency>
-        <dependency>
-            <groupId>commons-lang</groupId>
-            <artifactId>commons-lang</artifactId>
-        </dependency>
+		<dependency>
```
Comment (buckett) [inline polls/tool/pom.xml]: Most of the rest of the file looks to use spaces rather than tabs.

File: reset-pass/reset-pass/pom.xml [hunk 3]
```diff
@@ -68,9 +68,9 @@
       <type>war</type>
     </dependency>
 <!-- Commons JARS put in shared/lib by Sakai -->
-    <dependency>
-      <groupId>commons-lang</groupId>
-      <artifactId>commons-lang</artifactId>
-    </dependency>
+	<dependency>
```
Comment (buckett) [inline reset-pass/reset-pass/pom.xml]: Indentation looks different again to the rest of the file.

**Comment by axxter99** [comment]
StringEscapeUtils.unescapeHtml3() (commons-lang3 http://commons.apache.org/proper/commons-lang/javadocs/api-release/org/apache/commons/lang3/StringEscapeUtils.html)?

**Comment by axxter99** [comment]
StringEscapeUtils.unescapeHtml3() (commons-lang3 http://commons.apache.org/proper/commons-lang/javadocs/api-release/org/apache/commons/lang3/StringEscapeUtils.html)?

**Pull request merged**